server/lib/JJ.pm

```diff
@@ -0,0 +1,31 @@
+package JJ;
+
+use strict;
+use warnings;
+
+use Data::Dumper;
+
+our $VERSION = '0.1';
+
+=head1 NAME
+
+JJ
+
+=head1 DESCRIPTION
+
+Simple debugging helper that logs in BBEdits Logs directory.
+
+=cut
+
+sub jjlog {
+    my ( $sender, $object ) = @_;
+    my $log_file_path =
+        "~/Library/Containers/com.barebones.bbedit/Data/Library/Logs/BBEdit/perl-PLS.log";
+    open( FH, '>>', glob( $log_file_path ) ) or die $!;
```

Comment:
Bareword filehandles are not acceptable in this codebase. You must use a lexical filehandle.

Additionally, my style dictates that open should not have parentheses.

```suggestion
    open my $fh, '>>', glob($log_file_path) or die $!;
```

Reply:
Noted